src/apps/inventory/views/Forms/BulkWorkOrder/index.jsx

```diff
@@ -355,7 +355,7 @@ function Configurator({ openUserTunnel, openStationTunnel, bulkWorkOrder }) {
    return (
       <>
          <Separator />
-         <Flex width="10rem">
+         <Flex width="10rem" style={{ marginLeft: '16px' }}>
```

Comment:
Use `Flex`'s own `margin` prop:
```suggestion
         <Flex width="10rem" margin="0 0 0 16px"}>
```
Make this change everywhere you've used `style={{}}`